### ISSUE TITLE
"Interesting" output for annexed files

### ISSUE THREAD

**Issue by Remi-Gau** [incf-nidash/PyNIDM#320]
Working on a dataset where some text files were annexed with datalad.

In this case `bidsmri2nidm` seems to go for the annexed content rather than giving something human readable

```bash
You will now be asked a series of questions to annotate your term: 
/annex/objects/MD5E-s115--0ccac146d107ecea40039e0fb1e1806c.tsv
```

**Comment by dbkeator** [comment]
Hi,
I'm not sure of the distinction.  I'm not much of a datalad power user.

We added this functionality consulting with Yarik.  I'm sure things have changed.  Here's the code where this gets set: https://github.com/incf-nidash/PyNIDM/blob/master/nidm/experiment/Utils.py#L2098

**Comment by yarikoptic** [comment]
@Remi-Gau didn't look inside or in detail yet, but as you might have learned since then -- this smells like git-annex committed a .tsv file in unlocked form, so whenever it is not present - it is a text file with that content, but then when `get`-ed - would be a regular text file with target content.

**Comment by Remi-Gau** [comment]
yeah would need to double check... I have not gone back to this in a while though: It has been put on a back burner that was itself moved into the attic...
